sympy/sets/sets.py

```diff
@@ -1298,8 +1298,8 @@ def identity(self):
     def zero(self):
         return S.UniversalSet
 
-    def __new__(cls, *args, **kwargs):
-        evaluate = kwargs.get('evaluate', global_parameters.evaluate)
+    def __new__(cls, *args,evaluate=global_parameters.evaluate):
```

Comment:
Their should be a space after a comma.

Reply:
ok will do it just now

Reply:
Now that I have already created this PR , should I delete the older one?

Reply:
This is my first contribution to open source . Can someone guide me why the PR does not pass all the tests . I am not able to see a green tick against my PR.